src/languages/java.md

```diff
@@ -1,3 +1,101 @@
 # Java
 
-Java support is in Alpha and not generally available. Contact sales [https://platform.sh/contact](https://platform.sh/contact) if you are interested in running Java based applications on Platform.sh.
+Java is the most popular language in the globe, and it is in several fields:
+
+* Mobile applications
```

Comment:
Is this list necessary here? We don't have something similar for the other languages, just a one or two sentence description.

Reply:
better?